src/olympia/reviewers/utils.py

```diff
@@ -1452,16 +1452,8 @@ def approve_latest_version(self):
         log.info('Sending email for %s' % (self.addon))
 
     def block_multiple_versions(self):
-        min_version = ('0', None)
-        max_version = ('*', None)
-        for version in self.data['versions']:
-            version_str = version.version
-            if not min_version[1] or version_str < min_version[1]:
-                min_version = (version, version_str)
-            if not max_version[1] or version_str > max_version[1]:
-                max_version = (version, version_str)
-
-        params = f'?min={min_version[0].pk}&max={max_version[0].pk}'
+        versions = self.data['versions']
+        params = f'?{"&".join(f"v={v.id}" for v in versions)}'
```

Comment:
Doesn't matter much in this case, but for consistency let's try to avoid dealing with query parameters manually:

```suggestion
        params = f'?{urlencode(("v", v.id) for v in versions)}'
```

(Or `'?' + ... `. Gotta import `urlencode` from `django.utils.http` - the `urllib.parse` variant doesn't accept a generator directly, would have to convert into a list first)

Reply:
https://github.com/mozilla/addons-server/pull/20837/commits/89c8cad035fa31c00304f8f0aad7c3080cbcf3a7